aleph-client/src/session.rs

```diff
@@ -137,3 +138,22 @@ pub fn wait_for_at_least<C: AnyConnection>(
 pub fn get_session_period<C: AnyConnection>(connection: &C) -> u32 {
     connection.read_constant("Elections", "SessionPeriod")
 }
+
+pub fn get_authorities_for_session<C: AnyConnection>(
+    connection: &C,
+    session: u32,
+    session_period: u32,
+) -> Vec<AccountId> {
+    let first_block = session_period * session;
+    let block = get_block_hash(connection, first_block);
+
+    connection
+        .as_connection()
+        .get_storage_value("Session", "Validators", Some(block))
+        .expect("Failed to decode Validators extrinsic!")
+        .expect("Authorities should always be present")
+}
+
+pub fn get_current_validators<C: AnyConnection>(connection: &C) -> Vec<AccountId> {
```

Comment:
It's not consistently named compared with previous method: something like `get_current_authorities`.

Reply:
Right, changed to make it more consistent.